ingest/scripts/call_loculus.py

```diff
@@ -426,24 +427,10 @@ def get_sequence_status(config: Config):
     return result
 
 
-def get_submitted(config: Config):
-    """Get previously submitted sequences
+def get_submitted(config: Config, output: str):
```

Comment:
Might be nice to add return types and/or the shape of the output in the doc comment